parcels/particleset.py

```diff
@@ -160,6 +111,7 @@ def ArrayClass_init(self, *args, **kwargs):
             time = np.array([np.datetime64(t) for t in time])
         if time.size > 0 and isinstance(time[0], np.timedelta64) and not self.time_origin:
             raise NotImplementedError("If fieldset.time_origin is not a date, time of a particle must be a double")
+
```

Comment:
The error message in the line above is not true anymore, right? Particle time cannot be a double?

Reply:
You're right. I'll get this on my next round of edits

Reply:
Resolved in https://github.com/OceanParcels/Parcels/pull/2060/commits/a55c8520fd273d90ec9e86c16bfbdfbdbc264d01